app/Http/Controllers/RecipeController.php

```diff
@@ -107,32 +99,29 @@ public function show(Recipe $recipe)
             $recipe->load('user');
 
             if (Auth::user()) {
```

Comment:
Check if user is authenticated should be done in middleware. Route protected with middleware